lib/core/CMakeLists.txt

```diff
@@ -338,7 +338,6 @@ install(
   "${CMAKE_CURRENT_SOURCE_DIR}/include/irods/irods_load_plugin.hpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/include/irods/irods_log.hpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/include/irods/irods_logger.hpp"
-  "${CMAKE_CURRENT_SOURCE_DIR}/include/irods/irods_logger.tpp"
```

Comment:
Should we delete the `irods_logger.tpp` file as well?

Reply:
done